index.html

```diff
@@ -38,11 +38,11 @@ <h1>Hey There.
             </div>
             
             <ul>
-                <li><a href="#"><img src="images/Github.png" alt="github-icon"></a> </li>
-                <li><a href="#"><img src="images/linkedin.png" alt="linked-icon"></a> </li>
-                <li><a href="#"><img src="images/chill.png" alt="chill-icon"></a> </li>
-                <li><a href="#"><img src="images/twitter.png" alt="twitter-icon"></a> </li>
-                <li><a href="#"><img src="images/Medium.png" alt="Medium-icon"></a> </li>
+                <li><a href="#"><img src="images/Github.png" alt="github-icon image"></a> </li>
+                <li><a href="#"><img src="images/linkedin.png" alt="linked-icon image"></a> </li>
+                <li><a href="#"><img src="images/chill.png" alt="chill-icon image"></a> </li>
+                <li><a href="#"><img src="images/twitter.png" alt="twitter-icon image"></a> </li>
+                <li><a href="#"><img src="images/Medium.png" alt="Medium-icon image"></a> </li>
```

Comment:
Kinldy remove the `image` word from the alt texts as described [here](https://www.w3.org/WAI/test-evaluate/preliminary/#images) 😃

![image](https://user-images.githubusercontent.com/73714615/188617547-9ed3a23f-5968-4985-89a5-abc412043895.png)